frontend/dockerfile/dockerfile_test.go

```diff
@@ -2935,15 +2958,22 @@ ENV foo=bar
 
 	require.NotEqual(t, "", ociimg.OS)
 	require.NotEqual(t, "", ociimg.Architecture)
-	require.NotEqual(t, "", ociimg.Config.WorkingDir)
 	require.Equal(t, "layers", ociimg.RootFS.Type)
-	require.Equal(t, 0, len(ociimg.RootFS.DiffIDs))
-
-	require.Equal(t, 1, len(ociimg.History))
-	require.Contains(t, ociimg.History[0].CreatedBy, "ENV foo=bar")
-	require.Equal(t, true, ociimg.History[0].EmptyLayer)
-
-	require.Contains(t, ociimg.Config.Env, "foo=bar")
+	if runtime.GOOS == "windows" {
+		require.Equal(t, 1, len(ociimg.RootFS.DiffIDs))
```

Comment:
nit: can still use `integration.UnixOrWindows` here instead of the `if runtime.GOOS`. For example
```go
require.Equal(t, integration.UnixOrWindows(1, 0), len(ociimg.RootFS.DiffIDs)
```